website/docs/guides/miscellaneous/Ignoring Functions.md

```diff
@@ -0,0 +1,22 @@
+# Ignoring Functions
```

Comment:
todo(fzyzcjy): maybe move this doc

(this is a remark for me, no worries, you can ignore it)